README.md

```diff
@@ -468,20 +468,26 @@ knpu_oauth2_client:
             # a route name you'll create
             redirect_route: connect_azure_check
             redirect_params: {}
-
-            # whether to check OAuth2 "state": defaults to true
-            # use_state: true
-
-            # Azure specific options
-            # url_login: 'https://login.microsoftonline.com/'
-            # path_authorize: '/oauth2/authorize'
+            # Domain to build login URL
+            # url_login: ''
```

Comment:
Same here - with `example()` I think we can still pre-fill the value rendered here - which is helpful